plugins/bc-custom-content/tests/TestCase/Model/Table/CustomEntriesTableTest.php

```diff
@@ -87,14 +91,75 @@ public function test_createSearchDetail()
     /**
      * test setUp
      */
-    public function test_setUp()
+    public function test_setUp1()
     {
         //準備
-
+        $dataBaseService = $this->getService(BcDatabaseServiceInterface::class);
+        $customTable = $this->getService(CustomTablesServiceInterface::class);
+        CustomFieldFactory::make([
+            'id' => 1,
```

Comment:
The id is probably not needed here, you may want to leave the id set automatically by the DB engine.
Generally, you may want to define [here](https://github.com/baserproject/basercms/blob/dev-5/plugins/bc-custom-content/tests/Factory/CustomFieldFactory.php#L44) all the defaul values and set here only the ones relevant for the test case. This will save you readability and maintenance :-) 
Glad to see you using the factories!

Reply:
thank you!